client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/upgrade-popover/style.scss

```
@@ -0,0 +1,33 @@
+.upgrade-popover {
+	border-radius: 4px;
+
+	.popover__inner {
+		display: flex;
+		gap: 16px;
+		padding: 16px;
+		flex-direction: column;
+		align-items: flex-start;
+	}
+}
+
+.upgrade-popover__heading {
+	font-size: rem(14px);
+	line-height: 1.5;
+}
+
+.upgrade-popover__list {
+	margin: 0;
+	list-style: none;
+	font-size: rem(14px);
+	text-align: left;
+	display: flex;
+	flex-direction: column;
+	gap: 2px;
+
+	li {
+		background: url(calypso/assets/images/jetpack/jetpack-green-checkmark.svg) no-repeat 0 2px;
+		padding-left: 24px;
```

Could we use the RTL equivalent here?

```suggestion
		padding-inline-start: 24px;
```